### PR TITLE
fundmover readme changes

### LINKED ISSUE #780: [Examples] Test FundMover example.
We have an example - FundMover - for testing TwoPCCoordinatorPolicy manually. Style format doesn't seem to be enforced. Test this app, confirm style format and Gradle restrictions are enforced and close this issue.

### PR DESCRIPTION
**Pull request by Vishwa4jeet**
Fixes #780 

1. Identified some readme changes.
2. Docker command has been improved to map docker port to host port using localhost ip.

Capture1

Capture2

### CODE REVIEW

File: examples/fundmover/README.md [hunk 1]
```diff
@@ -1,36 +1,53 @@
 # Fundmover Demo App
 
   App demonstrates the usage of TwoPC DM with Maria Database ,
-  BankAccount and Wallet class  extends the TwoPCExtResourceCohortPolicy DM and 
+  BankAccount and Wallet class extends the TwoPCExtResourceCohortPolicy DM and 
   Finance class extends the TwoPCCoordinatorPolicy DM. 
 
-#Setup 
-  Follow the Below step to build and configure the Maria DB
+#Setup
```
Comment (maheshrajus) [inline examples/fundmover/README.md]: suggestion: trail space

File: examples/fundmover/README.md [hunk 2]
```diff
@@ -1,36 +1,53 @@
...
-4. sudo docker exec -it mariadbdcap bash 
-5. mysql -uroot -p
-   use the password defined  in Step 2 
-6. Execute all  the query  present in sql-scripts/setup.sql file 
-7. Exit from the CLi and container 
+2. docker run --name mariadbamino -e MYSQL_ROOT_PASSWORD=mysecretpw -d mariadb/server:latest  
+In Place of mysecretpw specify the password to be used 
+3. docker start mariadbamino
+4. sudo docker exec -it mariadbamino bash 
+5. mysql -uroot -p
```
Comment (maheshrajus) [inline examples/fundmover/README.md]: suggestion: trail space

File: examples/fundmover/README.md (hunk 1 above)
Comment (Vishwa4jeet) [reply examples/fundmover/README.md]: Trailing space is used in markdown file for adding a newline.
https://stackoverflow.com/questions/33191744/how-to-add-new-line-in-markdown-presentation

File: examples/fundmover/README.md (hunk 2 above)
Comment (Vishwa4jeet) [reply examples/fundmover/README.md]: Trailing space is used in markdown file for adding a newline.
https://stackoverflow.com/questions/33191744/how-to-add-new-line-in-markdown-presentation

**Pull request merged**